### PR TITLE
Freeze plugins version for 1.5

### PR DESCRIPTION
**Pull request by purbon**
so we have the last released plugins.

### CODE REVIEW

**Comment by ph** [comment]
LGTM

File: Gemfile.jruby-1.9.lock.defaults [hunk 1]
```diff
@@ -241,7 +238,7 @@ GEM
     logstash-input-exec (0.1.3)
       logstash-codec-plain
       logstash-core (>= 1.4.0, < 2.0.0)
-    logstash-input-file (0.1.6)
+    logstash-input-file (0.1.7)
       addressable
       filewatch (= 0.6.1)
```
Comment (suyograo) [inline Gemfile.jruby-1.9.lock.defaults]: We recently updated filewatch to 0.6.2. can we reflect this?

**Comment by suyograo** [comment]
Minor comment re: filewatch gem version

File: Gemfile.jruby-1.9.lock.defaults (hunk 1 above)
Comment (ph) [inline Gemfile.jruby-1.9.lock.defaults]: Good catch
The PR was created after the release of the filewatch.
``
filewatch = 0.6.2
logstash-input-file  = 0.1.8
``

**Comment by suyograo** [comment]
Moved here: https://github.com/elastic/logstash/pull/3016

**Comment by purbon** [comment]
Makes sense! good this change is in :-)

**Pull request closed without merging**